docs/develop/auth.md

```diff
@@ -77,6 +76,32 @@ func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
 }
 ```
 
+## Handling auth errors
+
+When a token doesn't match your auth rules (for example if it's expired, the token has been revoked, or the token is invalid), you should return a non-nil error from the auth handler.
+
+Encore passes the error message on to the user when you use [Encore's built-in error package](errors), so we recommend using that with the error code `Unauthenticated` to communicate what happened. For example:
+
+```go
+import "encore.dev/beta/errs"
+
+//encore:authhandler
+func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
+    return "", &errs.Error{
+        Code: errs.Unauthenticated,
+        Message: "invalid token",
+    }
+}
+
```

Comment:
Remove these two empty lines; they're part of a code block so would show up in the fully rendered doc.

Reply:
@eandre i have removed the empty lines